LayoutTest/TestCase/LYTLayoutTestCase.m

```diff
@@ -29,6 +31,16 @@ @implementation LYTLayoutTestCase
 
 #pragma mark - Running Tests
 
++ (void)setUp {
```

Comment:
Yeh the class method version is called at the start of the test class being run before any tests are run. Similarly `+(void)tearDown;` is called after all the tests in the class have run.